pkg/xds/secrets/secrets.go

```diff
@@ -339,7 +339,10 @@ func (s *secrets) generateCerts(
 
 			otherCa, _, err := s.caProvider.Get(context.Background(), otherMesh)
 			if err != nil {
-				return nil, errors.Wrap(err, "could not get other mesh CA cert")
+				// The other CA is misconfigured but this can not affect
+				// generation in this mesh.
+				log.Error(err, "could not get other mesh CA cert")
+				continue
```

Comment:
so let's say we have a problem and we fail to acquire CA. How do we recover from this problem? Do we try to fetch it again on the next XDS reconciliation?

Reply:
If it's a misconfiguration that's corrected, it will cause a regeneration.
A temporary error? Good question, yes, it'd be the next reconciliation.

Reply:
I've changed this so at the next reconciliation, the CP will attempt to regenerate certs if one of the other mesh CAs fails.

Reply:
nice!